make/StaticLibsImage.gmk

```diff
@@ -49,6 +49,16 @@ $(foreach m, $(ALL_MODULES), \
   $(eval TARGETS += $$(COPY_STATIC_LIBS_$m)) \
 )
 
+$(foreach v, $(JVM_VARIANTS), \
+  $(eval $(call SetupCopyFiles, COPY_STATIC_LIBS_libjvm, \
+      SRC := $(HOTSPOT_OUTPUTDIR)/variant-$v/libjvm/objs/static, \
+      DEST := $(STATIC_LIBS_IMAGE_DIR)/lib, \
+      FILES := $(filter %$(STATIC_LIBRARY_SUFFIX), \
+          $(call FindFiles, $(HOTSPOT_OUTPUTDIR)/variant-$v/libjvm/objs/static)), \
+  )) \
+  $(eval TARGETS += $$(COPY_STATIC_LIBS_libjvm)) \
+)
+
```

Comment:
Won't this fail if we were building more than one variant? Perhaps only put `$(JVM_VARIANT_MAIN)` in the static image to avoid problems?

Reply:
Changed to copy for $(JVM_VARIANT_MAIN) only, as suggested. Thanks.